CHANGELOG.md

```
@@ -2,6 +2,27 @@
 
 All notable changes to cmux are documented here.
 
+## [0.64.2] - 2026-05-05
+
+### Fixed
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)